src/doc/version-switcher.js

```diff
@@ -0,0 +1,269 @@
+(function() {
+
+let CURRENT_VERSION = -1;
```

Comment:
Feels like this should be null to be a better indication that it's a sentinel value.